### DISCUSSION TITLE
模组切换界面

### DISCUSSION THREAD

**Discussion by 0kk470** [jynew/jynew#1297] [Ideas]
**功能描述**

用于动态切换模组，以及一些简单的管理。

**UI导航路径**

只需要方向导航以及确认、关闭操作。

![image](https://user-images.githubusercontent.com/25216715/199189330-a21cacc2-0f61-4fb4-97fb-81e471092994.png)

**Comment by qige2016** [comment]
这个好像只有切换MOD和关闭做了

**Reply by 0kk470** [reply]
我到这的时候补一下功能

**Reply by qige2016** [reply]
先做这两个吧，这个页面计划是把创意工坊页面也集成进来

**Reply by 0kk470** [reply]
> 先做这两个吧，这个页面计划是把创意工坊页面也集成进来

暂时把MOD商店隐藏掉了，其他几个按钮的功能都加了
